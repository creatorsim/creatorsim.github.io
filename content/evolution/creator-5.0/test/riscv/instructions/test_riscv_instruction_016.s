
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:
	li x5, 752
    li x10, -1024
    
	sltiu x6, x5, 751
    sltiu x7, x5, 752
    sltiu x8, x5, 753
    sltiu x9, x5, 754
    
    sltiu x11, x10, -1025
    sltiu x12, x10, -1024
    sltiu x13, x10, -1023
    sltiu x14, x10, -1022
    