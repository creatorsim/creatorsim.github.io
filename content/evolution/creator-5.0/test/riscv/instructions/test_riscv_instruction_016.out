
cr[PC]:0x28; ir[x5,t0]:0x2f0; ir[x8,fp,s0]:0x1; ir[x9,s1]:0x1; ir[x10,a0]:0xfffffc00; ir[x13,a3]:0x1; ir[x14,a4]:0x1; keyboard[0x0]:''; display[0x0]:''; 

